behaviors/wysiwyg.js

```diff
@@ -440,7 +440,7 @@ function initWysiwygBinder(enableKeyboardExtras) {
         // SelectionAPI R'lyeh fhtagn
         range = document.createRange();
         selection = window.getSelection();
-        range.setStart(editable, 1);
+        range.setStartAfter(editable.lastChild);
```

Comment:
`Node#lastChild` returns the last child (not just the last element) of a node. This is what we want :-)